### PR TITLE
Add `renditionCount` and `totalRenditionsSize` metrics to activation metrics

### PR DESCRIPTION
**Pull request by jdelbick**
## Description

As we start to work on optimization, we may find the need to look at historical data in new relic to make informed decisions. I noticed a few missing metrics that could help, so I am adding them now so we can start gathering data.

`renditionCount`: number of renditions attempted for processing
- this was only available in `process` metrics until now. If we look at action duration without knowing the amount of renditions, our decisions may have high variance. The number of renditions has a large impact on the action duration.

`totalRenditionsSize`: sum of the size of each rendition successfully produce
- I thought it would be interesting to track this. It is highly related to rendition count, but I see it as a helpful thing to look for, especially in workers that produce larger renditions like transcoding

The idea is if we have all the metrics we need in one event type, we can more easily analyze it. New Relic does not have a `join` clause to merge data between event types (ie no way to merge rendition and activation metric types on the same activation id).

Another interesting metric to add to activation metrics could be `renditionErrorCount`
### Testing
- added to unit test metrics check
- checked metrics from IT test integration: https://onenr.io/0BoQDZ084Ry
   - note: still need to duplicate the changes in pipeline

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by bobvanmanen — APPROVED** [review]
LGTM

**Review by pheenomenon — APPROVED** [review]

**Review by tmathern — APPROVED** [review]

File: lib/worker.js [hunk 1]
```diff
@@ -392,6 +394,10 @@ class AssetComputeWorker {
 
         rendition.eventSent = true;
 
+        const renditionSize = rendition.size();
+        // track total rendition size to add to activation metrics
+        this.totalRenditionsSize += renditionSize;
```
Comment (tmathern) [inline lib/worker.js:399]: Only note: For large renditions, if there are a lot (a lot) of them, this can overflow.

Comment (jdelbick) [reply lib/worker.js:399]: true, I will create a safe add function that takes into account the max nodejs number: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

fwiw, the max number is `9007199254740991`, which would translate to >9007 terabytes. I am pretty sure nui would timeout first if we were uploading that large amount of terabytes

Comment (jdelbick) [reply lib/worker.js:399]: buff-overflow

Comment (tmathern) [reply lib/worker.js:399]: Right, but adding the check won't hurt.

Comment (bobvanmanen) [reply lib/worker.js:399]: It's for metrics, i don't think we need to go too far. If we have renditions this large we can always improve.

Comment (jdelbick) [reply lib/worker.js:399]: Ok I will not add this check now, but we can always address it if we are seeing large amounts of renditions or very large renditions

**Pull request merged**